src/providers/provider.ts

```diff
@@ -44,15 +44,24 @@ export class Provider extends EventEmitter {
         if (mock.shouldThrow) {
           reject(mock.data);
         } else {
-          resolve({ data: mock.data, callback: mock.triggerCallback });
+          resolve({
+            data: mock.data,
+            callback: mock.triggerCallback,
+          });
         }
       }, mock.timeout || 0);
     });
     const { data, callback } = await promise;
+    let returnValue: unknown = undefined;
```

Comment:
[Optional]: `let returnValue: unknown;` is enough, the `undefined` assignment makes me think that this value is used without future assignment 😁